install_ntf.sh

```diff
@@ -18,10 +18,11 @@ install_ntf() {
 		doxygen \
 		libjansson-dev \
 		libjansson4 \
-		libtool
+		libtool \
+		unzip
```

Comment:
can remove the need for unzip like shown here

https://github.com/omec-project/upf-epc/blob/e471cef3161e716834510477a6fa61844ae8afb8/Dockerfile#L16-L29

Reply:
Done!